generate/src/main.rs

```diff
@@ -27,6 +27,7 @@ fn main() {
     let matches = command!()
         .args(&[
             arg!(-d --debug "generate a program where values are printed instead of hashed (slow)"),
+            arg!(-p --printf_debug "generate a program where values are printed using the C 'printf' function instead of hashed (slow)"),
```

Comment:
CLI stability is not a concern. So you could replace `-d` with a single enum flag with 3 options, defaulting to `HashDumper`